scripts/steam.zs

```diff
@@ -1,33 +1,134 @@
+		mods.immersivetechnology.HeatExchanger.addRecipe(transferOut[i] * (num*transferModifier[i]), coolantOut[j] * coolModifier[i],  transferIn[i] * num, coolantIn[j] * (coolModifier[i]), 0, 1);
+	}
+}
+
+/* -=-=-=-=-=-=-=-=- Moar Heat Sinks Integration -=-=-=-=-=-=-=-=- */
+if (oreDict has "QuantumTraverseWouldYouKindlyStuffASockInIt") {
+
+crMatrix = [[625,1075,1125,950,1100,350,800,875,725,875,550,600,725,900,1200,2815,150],[625,1075,1125,950,1100,350,800,875,725,875,550,600,725,900,1200,2815,150],[2500,4300,4500,3800,4400,1400,3200,3500,2900,3500,2200,2400,2900,3600,4800,11260,600],[3125,5375,5625,4750,5500,1750,4000,4375,3625,4375,2750,3000,3625,4500,6000,14075,750],[375,645,675,570,660,210,480,525,435,525,330,360,435,540,720,1689,90],[3125,5375,5625,4750,5500,1750,4000,4375,3625,4375,2750,3000,3625,4500,6000,14075,750]] as int[][];
+coolantIn = [<liquid:steel_nak_hot>,<liquid:ferroboron_nak_hot>,<liquid:tough_alloy_nak_hot>,<liquid:limno2_nak_hot>,<liquid:mgb2_nak_hot>,<liquid:boron_arsenide_nak_hot>,<liquid:boron_nitride_nak_hot>,<liquid:rhodochrosite_nak_hot>,<liquid:zirconium_nak_hot>,<liquid:hard_carbon_nak_hot>,<liquid:extreme_alloy_nak_hot>,<liquid:thermoconducting_alloy_nak_hot>,<liquid:zircaloy_nak_hot>,<liquid:sic_sic_cmc_nak_hot>,<liquid:hsla_nak_hot>,<liquid:smore_nak_hot>,<liquid:liquid_oxygen_nak_hot>] as ILiquidStack[];
+coolantOut = [<liquid:steel_nak>,<liquid:ferroboron_nak>,<liquid:tough_alloy_nak>,<liquid:limno2_nak>,<liquid:mgb2_nak>,<liquid:boron_arsenide_nak>,<liquid:boron_nitride_nak>,<liquid:rhodochrosite_nak>,<liquid:zirconium_nak>,<liquid:hard_carbon_nak>,<liquid:extreme_alloy_nak>,<liquid:thermoconducting_alloy_nak>,<liquid:zircaloy_nak>,<liquid:sic_sic_cmc_nak>,<liquid:hsla_nak>,<liquid:smore_nak>,<liquid:liquid_oxygen_nak>] as ILiquidStack[];
+val coolModifier = [144,64,6,60,540,80] as int[];
+var transferIn = [<liquid:water>,<liquid:preheated_water>, <liquid:exhaust_steam>,<liquid:condensate_water>,<liquid:well_pressed_water>,<liquid:crap_steam>] as ILiquidStack[];
+var transferOut = [<liquid:high_pressure_steam>,<liquid:high_pressure_steam>,<liquid:low_pressure_steam>,<liquid:preheated_water>,<liquid:super_critical_steam>,<liquid:high_pressure_steam>] as ILiquidStack[];
+var transferModifier = [4,4,1,1,1,1] as int[];
+
+//MHS-NCO HX recipes
+for i,array in crMatrix{
+	for j,num in array{
+		mods.nuclearcraft.ChemicalReactor.removeRecipeWithInput(coolantIn[j] * coolModifier[i], transferIn[i] * num);
+		mods.immersivetechnology.HeatExchanger.addRecipe(transferOut[i] * (num*transferModifier[i]), coolantOut[j] * coolModifier[i],  transferIn[i] * num, coolantIn[j] * (coolModifier[i]), 0, 1);
+	}
+}
+
+mods.nuclearcraft.ChemicalReactor.removeRecipeWithInput(<liquid:corium_nak_hot> * 60, <liquid:preheated_water> * 125);
+mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:corium_nak>*60, <liquid:condensate_water>*125, <liquid:corium_nak_hot>*60, <liquid:preheated_water>*125, 0, 1);
```

Comment:
These lines appear to be duplicates and should be removed. Also that would prevent an error with the chemical reactor since it is trying to remove a recipe twice.